documentation/install_from_packages.markdown

```diff
@@ -1,16 +1,27 @@
 ---
 layout: default
-title: "Puppet Server: Installing From Packages"
+title: "Install Puppet Server"
 canonical: "/puppetserver/latest/install_from_packages.html"
 ---
 
-[repodocs]: https://puppet.com/docs/puppet/latest/puppet_platform.html
+Puppet Server is a required application that runs on the Java Virtual Machine (JVM). It performs the role of the master node that controls configuration information for one or more managed agent nodes
 
-Puppet Server is configured to use 2 GB of RAM by default. If you're just testing an installation on a Virtual Machine, this much memory is not necessary. To change the memory allocation, see [Memory Allocation](#memory-allocation).
+> Note: If you have any issues with the steps below, submit these to our [bug tracker](https://tickets.puppet.com/browse/SERVER).
 
-> If you're also using PuppetDB, check its [requirements](https://puppet.com/docs/puppetdb/latest/index.html#system-requirements).
+## Before you begin
 
-## Java support
+Review the system requirements and make sure you have a supported version of Java. 
```

Comment:
What are you referring to when you say 'system requirements'? Do you just mean the supported operating systems below? Or another section we should link to? I think this could be a little clearer.